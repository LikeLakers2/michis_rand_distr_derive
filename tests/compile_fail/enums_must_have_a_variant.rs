use michis_rand_distr_derive::StandardDistribution;
fn main() {}

#[derive(StandardDistribution)]
pub enum ErrorTest {}
