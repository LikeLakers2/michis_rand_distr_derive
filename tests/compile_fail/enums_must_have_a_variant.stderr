error: proc-macro derive panicked
 --> tests/compile_fail/enums_must_have_a_variant.rs:4:10
  |
4 | #[derive(StandardDistribution)]
  |          ^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Cannot derive StandardDistribution for enums with zero variants
